test/test-postgresql-bindings/mappedDataBindings/index.js

```diff
@@ -38,8 +38,6 @@ const crunchyDB = {
     '# Generated by postgres-operator. DO NOT EDIT.\n# Your changes will not be saved.\n\n[global]\nlog-path = /pgbackrest/repo1/log\nrepo1-path = /pgbackrest/repo1\n\n[db]\npg1-host = hippo-instance1-87wl-0.hippo-pods.postgres-operator.svc.cluster.local.\npg1-host-ca-file = /etc/pgbackrest/conf.d/~postgres-operator/tls-ca.crt\npg1-host-cert-file = /etc/pgbackrest/conf.d/~postgres-operator/client-tls.crt\npg1-host-key-file = /etc/pgbackrest/conf.d/~postgres-operator/client-tls.key\npg1-host-type = tls\npg1-path = /pgdata/pg14\npg1-port = 5432\npg1-socket-path = /tmp/postgres',
   port: '5432',
   provider: 'crunchydata',
-  'root.crt':
```

Comment:
I'm not sure if we should remove this attribute. If we add it the tests fail due to root.crt gets overridden by ca.crt. We need to find a solution according to the provider (binding data) or at least validate that with root.crt the pg client will be able to connect. I'm investigating.

Reply:
fixed the others except this one, let me know what I should do.

Reply:
@myeung18  Which third party psql DB service do you use?

Reply:
I guess your db is hosted on https://cockroachlabs.cloud

Reply:
I'll investigate tomorrow because I didnt manage to install the rhoda operator. Probably we have to go with an implementation of creating the mappings according to the provider. I have to discuss about that also with @mhdawson .
We really appreciate for taking the time to contribute on kube-service-bindings. We will let you know asap on how should we tackle this :)

Reply:
I used mainly RHODA. It helped to provision a few 3rd party DB instances such as psql and mysql from RDS, and instances cockroachdb cloud (https://cockroachlabs.cloud/).

Reply:
@myeung18
By connecting RHODA with cockroachdb I see two things
* The file generated on the bindings is called username instead of user
* There is a file called provider which has the value `CockroachDB Cloud`
Can you validate that, in order to properly add/rename the files under `test/test-postgresql-bindings/bindings/postgresql-rhoda`

Also are the test/test-postgresql-bindings/bindings/postgresql-rhoda-opt generated by cockroachdb or are generated from another third party provider?

Reply:
you're correct. I did a new push for `username` and `provider` files under those two test cases.

The /bindings folder is generated by SBO; by drawing a line between a CrdbDBaaSConnection object and a Node.js App. in the developer view of the OCP.
The CrdbDBaaSConnection object is part of / managed by cockcoachdb operator and it is also integrated with RHODA. This connection object contains all the credential and connection infos of a particular Crdb instance for SBO to do the bindings.  Hope this answers the question.

Reply:
The procedure you described is for the /bindings/postgresql-rhoda
How the /bindings/postgresql-rhoda-opt is generated? Is other third party service except rhoda+cockroachdb cloud?

Reply:
In the actual environment, there is no fixed name for that folder under the /bindings. It is generated by SBO by looking at some properties like CrdbDBaaSConnection object name etc?

e.g.:
```
sh-4.4$ find bindings
bindings
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/..data
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/sslmode
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/root.crt
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/database
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/provider
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/port
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/option-cluster
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/type
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/password
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/username
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/sslrootcert
bindings/crdb-quarkus-app-d-a-single-cluster-aws-cfa56923e7-dbsc/host
```

Reply:
I dont see this file **option-cluster** somewhere in the bindings generate by the cockroachdb Instead I see the **options** file, this is something we should handle in case of is the same thing but with different name. Did you choose to create a serverless or a dedicated cluster on this page https://cockroachlabs.cloud/cluster/create ?

Reply:
It is an example I meant to show you the folder name is dynamic generated by SBO, ignore the bindings content, they are not for CRDB.

I dont' use that page to create crdb at all.  Everything is done through RHODA and cockroachDB Operator. But now RHODA only creates serverless crdb cluster.